### DISCUSSION TITLE
Dec 2023 Community Call

### DISCUSSION THREAD

**Discussion by expede** [ucan-wg/spec#176] [Community Call]
# Agenda

- @expede
  - Thank you all for patience! 🙏
  - 1.0.0-rc.1 flipped to ready-for-review 🎉
    - Now the real work begins 😛 
    - Expect Rust/Wasm implementations of 1.0.0-rc.1 the first few weeks of 2024
  - Open questions solved:
    - "Idiot-proofed" the signature wrapper ✅
      - Will extract the signature container to own spec... later
    - Possible to encode as a JWT ✅ (via varsig)
    - Formats: delegation, invocation, receipt
      - All could be described as invocations with special nested fields
        - Team exploration: low ROI, adds complexity, add bytes
      - Revocations *are* a type of invocation
      - Adjusted invocation to work for attestations (`aud` not required, `sub` is required)

# Recording

# Chat Transcript

**Comment by gobengo** [comment]
Could we discuss https://github.com/ucan-wg/delegation/pull/2#discussion_r1423537864 ?
